core/whistle-1.0.0/src/wh_cache.erl

```diff
@@ -143,6 +145,11 @@ start_link(Name, ExpirePeriod, Props) ->
                                    )
     end.
 
+-spec maybe_add_db_binding(wh_proplists()) -> wh_proplists().
+maybe_add_db_binding([[]]) -> [[]];
```

Comment:
I commented line with `[[]]` to force bindings `[{conf,[federate,{type,<<"database">>}]},{conf,[federate]}]` which AFAIR is dont work for me (at first my try with wh_cache).
And i'm arguing about this binding.

Ddd `maybe_add_db_binding([]) -> [];` is not problem.